src/objects/tests/v2/test_validation.py

```diff
@@ -31,6 +31,61 @@ def setUpTestData(cls):
             token_auth=cls.token_auth,
         )
 
+    def test_valid_create_object_check_cache(self, m):
+        mock_service_oas_get(m, OBJECT_TYPES_API, "objecttypes")
+        m.get(
+            f"{self.object_type.url}/versions/1",
+            json=mock_objecttype_version(self.object_type.url),
+        )
+        m.get(
+            f"{self.object_type.url}/versions/2",
+            json=mock_objecttype_version(self.object_type.url),
+        )
+
+        url = reverse("object-list")
+        data = {
+            "type": self.object_type.url,
+            "record": {
+                "typeVersion": 1,
+                "data": {"plantDate": "2020-04-12", "diameter": 30},
+                "startAt": "2020-01-01",
+            },
+        }
+        with self.subTest("ok_cache"):
+            self.assertEqual(m.call_count, 0)
+            self.assertEqual(Object.objects.count(), 0)
+            for n in range(5):
+                self.client.post(url, data, **GEO_WRITE_KWARGS)
+            # just one request should run — the first one
+            self.assertEqual(m.call_count, 1)
+            self.assertEqual(Object.objects.count(), 5)
+
+        with self.subTest("clear_cache"):
+            m.reset_mock()
+            self.assertEqual(m.call_count, 0)
+            for n in range(5):
+                self._clear_caches()
+                self.client.post(url, data, **GEO_WRITE_KWARGS)
+            self.assertEqual(m.call_count, 5)
+            self.assertEqual(Object.objects.count(), 10)
```

Comment:
I'd prefer a test here that doesn't explicitly clear the cache, but instead uses `freezegun.freeze_time`:

```python
with freeze_time(<some_timestamp>):
  # create two objects
  # assert only one GET request was done
  # reset mock

with freeze_time(<some_timestamp+timeout>):
  # create another object
  # assert one GET request was done

```